### PR TITLE
[CUBRIDMAN-84] Manual for unloaddb.sh

### PR DESCRIPTION
**Pull request by kisoo-han**
http://jira.cubrid.org/browse/CUBRIDMAN-84

**Purpose**
unloaddb.sh which supports multi-process

**Implementation**
N/A

**Remarks**
New Section is created in "CUBRID Management"->"cubrid script"
Please refer following
- [link ko](http://192.168.2.38/ko/admin/scripts.html#unloaddb-sh-script)'
- [link en](http://192.168.2.38/en/admin/scripts.html#unloaddb-sh-script)'

### CODE REVIEW

**Review by airnet73 — APPROVED** [review]

File: en/admin/scripts.rst [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    The following example shows an input file (table_list.txt). ::
+
+        table_1
+        table_2
+        ..
+        table_n
+
+.. option:: -D output-directory
+
+    Specifies the directory in which to create schema and object files. If not specified, files are created in the current directory. ::
```
Comment (charischau) [inline en/admin/scripts.rst]:
```suggestion
    Specifies the directory in which to create schema and object files. If it is not specified, files are created in the current directory. ::
```

Comment (kisoo-han) [reply en/admin/scripts.rst]: Thank you for your comment!

**Review by charischau — APPROVED** [review]

**Review by airnet73 — APPROVED** [review]

File: en/admin/scripts.rst [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+	**restrictions:**
+	  * Require Linux Environments
+	  * Only users with **DBA privileges** can run this script
+
+The following is [options] used in **unloaddb.sh**.
+
+.. program:: sh unloaddb.sh
+
+.. option:: -u user
```
Comment (mhoh3963) [inline en/admin/scripts.rst:32]: At warning section, you wrote that this shell script is executed by only dba.
So, I think that the -u option don't need.
How about your think ?

File: en/admin/scripts.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+    Specify a user account of a database to be unloaded. If this is not specified, the default is **DBA**. ::
+
+        sh unloaddb.sh -u dba demodb
+
+.. option:: -t no-process
+
+    Number of child processes to run concurrently (default: 8, maximum 16) ::
+
+        sh unloaddb.sh -4 demodb
```
Comment (mhoh3963) [inline en/admin/scripts.rst]:
```suggestion
        sh unloaddb.sh -t 4 demodb
```

File: en/admin/scripts.rst [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+.. option:: -i input-class-file
+
+    Unload schemas, indexes and data of specified tables in the file. (default: all tables in the database) ::
+
+        sh unloaddb.sh -i table_list.txt demodb
+
+    The following example shows an input file (table_list.txt). ::
+
+        table_1
```
Comment (mhoh3963) [inline en/admin/scripts.rst]: From v11.2, cubrid supported user schema feature.
So, you need to replace table_1 to user.table_1 at the manual.
If it haven't user name in the table name, it is need to display error at v11.2 or later.
please check the -i option of unloaddb at v11.2.

File: ko/admin/scripts.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+    언로딩할 데이터베이스의 사용자 계정을 지정한다. 옵션을 지정하지 않으면 기본값은 DBA가 된다. ::
+
+        sh unloaddb.sh -u dba demodb
+
+.. option:: -t no-process
+
+    동시에 실행될 자식 프로세스의 개수 (no-process, 기본: 8개, 최대 16개 ) ::
+
+        sh unloaddb.sh -4 demodb
```
Comment (mhoh3963) [inline ko/admin/scripts.rst]:
```suggestion
        sh unloaddb.sh -t 4 demodb
```

File: en/admin/scripts.rst (hunk 2 above)
Comment (kisoo-han) [reply en/admin/scripts.rst:32]: Users belonging to the DBA group can also execute this script, even they are not 'dba' user. Not a user 'dba' but who has **DBA privileges**.

File: en/admin/scripts.rst (hunk 3 above)
Comment (kisoo-han) [reply en/admin/scripts.rst]: Thank you for your comment!

File: en/admin/scripts.rst (hunk 2 above)
Comment (mhoh3963) [reply en/admin/scripts.rst:32]: OK, I got it.
Then, It seems that it should be marked as a user with dba privileges.

Comment (kisoo-han) [reply en/admin/scripts.rst:32]: In the warning section, "Only users with **DBA privileges** can run this script", isn't it enough?

File: en/admin/scripts.rst (hunk 4 above)
Comment (kisoo-han) [reply en/admin/scripts.rst]: Thank you for your comment, I missed '**user schema**' !

File: en/admin/scripts.rst (hunk 2 above)
Comment (mhoh3963) [reply en/admin/scripts.rst:32]: You should write this mention at the -u option section too.

File: ko/admin/scripts.rst (hunk 5 above)
Comment (kisoo-han) [reply ko/admin/scripts.rst]: Thank you for your comment, it was fixed !

File: en/admin/scripts.rst (hunk 4 above)
Comment (kisoo-han) [reply en/admin/scripts.rst]: It was fixed.

File: en/admin/scripts.rst (hunk 2 above)
Comment (kisoo-han) [reply en/admin/scripts.rst:32]: You haven't approved it, can I proceed to merge?

**Review by mhoh3963 — APPROVED** [review]

**Pull request merged**